### PR TITLE
Change 'comment' hotkey to R instead of C

### LINKED ISSUE #859: Change 'comment' hotkey to R instead of C
Gmail uses R.

### PR DESCRIPTION
**Pull request by rd1992**
Fixes #859

### CODE REVIEW

File: src/main/java/ui/RepositorySelector.java [hunk 1]
```diff
@@ -49,6 +52,12 @@ private void setupComboBox() {
                 onValueChangeCallback.accept(repoId);
             }
         });
+        comboBox.addEventFilter(KeyEvent.KEY_RELEASED, event -> {
+            event.consume();
+            if (event.getCode() == KeyboardShortcuts.REMOVE_FOCUS) {
+                getParent().requestFocus();
+            }
+        });
```
Comment (damithc) [inline src/main/java/ui/RepositorySelector.java]: Perhaps add a comment to explain what this is doing?

File: src/test/java/guitests/ChromeDriverTest.java [hunk 2]
```diff
@@ -145,7 +145,7 @@ public void chromeDriverStubTest() {
         clearKeyCode();
 
         // jump to comments
-        push(KeyCode.C);
+        push(KeyCode.R);
```
Comment (dariusf) [inline src/test/java/guitests/ChromeDriverTest.java]: Same here, should make use of the constant.

File: src/main/java/ui/components/KeyboardShortcuts.java [hunk 3]
```diff
@@ -123,7 +126,7 @@ private static void addNonCustomizableShortcutKeys() {
         assignedKeys.add(KeyCode.F5); //REFRESH
         assignedKeys.add(KeyCode.F1); //SHOW_DOCS
         assignedKeys.add(KeyCode.G); //GOTO_MODIFIER
-        assignedKeys.add(KeyCode.C); //NEW_COMMENT
+        assignedKeys.add(KeyCode.R); //NEW_COMMENT
         assignedKeys.add(KeyCode.A); //MANAGE_ASSIGNEES
         assignedKeys.add(KeyCode.SPACE); //DOUBLE_PRESS
     }
```
Comment (dariusf) [inline src/main/java/ui/components/KeyboardShortcuts.java]: You could make use of the constants defined above here, then the comments wouldn't be needed.

**Comment by dariusf** [comment]
This doesn't have to depend on the Esc PR. Could you rebase onto master? Should also squash, since it's just a single change.

**Comment by dariusf** [comment]
Hmm, actually @damithc's comment should be addressed in #857. The squashing should be done there. This PR can just be 8aa3abb plus changes from feedback.

File: src/test/java/guitests/ChromeDriverTest.java (hunk 2 above)
Comment (dariusf) [inline src/test/java/guitests/ChromeDriverTest.java]: Realised that the constants aren't used much in tests. Unless we can `push` a `KeyCombination`...

**Comment by damithc** [comment]
remember to use labels to show current issue status.

**Comment by damithc** [comment]
Shall we wrap this up quickly?

**Comment by rd1992** [comment]
@dariusf i have squashed the commits in the other PR. You can merge this into master.

**Comment by dariusf** [comment]
Yes, but this still depends on those old commits and needs to be rebased as well. Like I said earlier, this PR should have only one commit (which you can squash changes based on feedback into). The easiest way to do this is to `rebase -i origin/master` and pick only the last one.

**Comment by dariusf** [comment]
Can we make use of the constants in `KeyboardShortcuts`? Can be merged otherwise.

**Comment by rd1992** [comment]
@dariusf we should use constants throughout. Though that should be a different issue than this no?

**Comment by dariusf** [comment]
Okay, sure.

**Comment by HansNewbie** [comment]
The keyboard shortcut docs has not been updated to use `R` instead of `C` for commenting.

**Comment by damithc** [comment]
Oops, should have been part of the same PR. @rd1992 please do as a separate issue ASAP. @darius, once it is done, we can simply merge to release branch. No need to do a new release because the documentation is online.

**Pull request merged**